<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateCollegeDialog</class>
 <widget class="QDialog" name="updateCollegeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update College</string>
  </property>
  <widget class="QLabel" name="updatecollegeCodeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>124</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COLLEGE CODE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updatecollegeNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COLLEGE NAME:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updcollegeCodeLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="updcollegeNameLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="updateCollegeButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="updatecollegeLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT COLLEGE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
